mojaloop-payment-token-adapter/templates/service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "mojaloop-payment-token-adapter.fullname" . }}
+  labels:
+{{ include "mojaloop-payment-token-adapter.labels" . | indent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    # todo: think, of we need this http port ?
+    - port: {{ .Values.service.httpPort }}
```

Comment:
I think we don't need this. We can use one of the two portal for health endpoint

Reply:
removed